lib/pages/settings_page.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:newyears_flutter/app_state_container.dart';
+import 'package:newyears_flutter/models/auth.dart';
+import 'package:newyears_flutter/models/user.dart';
+
+class SettingsPage extends StatefulWidget {
+  @override
+  SettingsState createState() {
+    return SettingsState();
+  }
+}
+
+class SettingsState extends State<SettingsPage> {
+  final _scaffoldKey = GlobalKey<ScaffoldState>();
+  User _user;
+
+  @override
+  Widget build(BuildContext context) {
+    final container = AppStateContainer.of(context);
+    _getUser();
+    return Scaffold(
+        appBar: AppBar(
+          title: Text('Settings'),
+        ),
+        key: _scaffoldKey,
+        body: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: <Widget>[
+            _user == null ? null : Text("First name: " + _user.firstName),
```

Comment:
We can try to avoid the duplicated condition

Reply:
I don't think there is a simpler way of checking null on the go in dart in this case (nothing like `_user ? x : y` in javascript). 
But this code will be deleted later on, when a proper layout will be built for this page anyways.